### PR TITLE
add new flag --last-start-only to only show last start logs 

### PR DESCRIPTION
**Pull request by nickmancari**
I closed my last PR. Here is a re-subbmission for a cleaner branch:

This PR adds the `minikube logs --last-start-only` flag feature as requested here: https://github.com/kubernetes/minikube/issues/15700

Before the PR:
```
$ minikube logs --last-start-only
Error: unknown flag: --last-start-only
See 'minikube logs --help' for usage.

```

After PR: 
```
$ minikube logs --last-start-only

==> Last Start <==
Log file created at: 2023/02/01 20:55:49
Running on machine: Nicks-MBP
Binary: Built with gc go1.19.5 for darwin/amd64
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
I0201 20:55:49.529075   63168 out.go:296] Setting OutFile to fd 1 ...
I0201 20:55:49.529464   63168 out.go:348] isatty.IsTerminal(1) = true
I0201 20:55:49.529469   63168 out.go:309] Setting ErrFile to fd 2...
I0201 20:55:49.529477   63168 out.go:348] isatty.IsTerminal(2) = true
I0201 20:55:49.529607   63168 root.go:334] Updating PATH: /Users/nick/.minikube/bin
W0201 20:55:49.529733   63168 root.go:311] Error reading config file at /Users/nick/.minikube/config/config.json: open /Users/nick/.minikube/config/config.json: no such file or directory
I0201 20:55:49.530772   63168 out.go:303] Setting JSON to false
I0201 20:55:49.573892   63168 start.go:125] hostinfo: {"hostname":"Nicks-MBP.home","uptime":5641601,"bootTime":1669664948,"procs":409,"os":"darwin","platform":"darwin","platformFamily":"Standalone Workstation","platformVersion":"12.6.1","kernelVersion":"21.6.0","kernelArch":"x86_64","virtualizationSystem":"","virtualizationRole":"","hostId":"2e97756e-e296-517d-96d1-91d34cd120f0"}
W0201 20:55:49.573997   63168 start.go:133] gopshost.Virtualization returned error: not implemented yet
I0201 20:55:49.579787   63168 out.go:177] 😄  minikube v0.0.0-unset on Darwin 12.6.1
I0201 20:55:49.588916   63168 notify.go:220] Checking for updates...
I0201 20:55:49.589715   63168 config.go:180] Loaded profile config "minikube": Driver=docker, ContainerRuntime=docker, KubernetesVersion=v1.26.1
...
```

### CODE REVIEW

**Review by medyagh — CHANGES_REQUESTED** [review]
thanks for resubmitting this PR, the After output shows the minikube logs command is still showing audit logs too... but it only should show the last start logs

**Comment by nickmancari** [comment]
> thanks for resubmitting this PR, the After output shows the minikube logs command is still showing audit logs too... but it only should show the last start logs

Oh I see. I corrected the output to only show the last start log:

```
$ minikube logs --last-start-only

==> Last Start <==
Log file created at: 2023/02/01 20:55:49
Running on machine: Nicks-MBP
Binary: Built with gc go1.19.5 for darwin/amd64
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
I0201 20:55:49.529075   63168 out.go:296] Setting OutFile to fd 1 ...
I0201 20:55:49.529464   63168 out.go:348] isatty.IsTerminal(1) = true
I0201 20:55:49.529469   63168 out.go:309] Setting ErrFile to fd 2...
I0201 20:55:49.529477   63168 out.go:348] isatty.IsTerminal(2) = true
I0201 20:55:49.529607   63168 root.go:334] Updating PATH: /Users/nick/.minikube/bin
W0201 20:55:49.529733   63168 root.go:311] Error reading config file at /Users/nick/.minikube/config/config.json: open /Users/nick/.minikube/config/config.json: no such file or directory
I0201 20:55:49.530772   63168 out.go:303] Setting JSON to false
I0201 20:55:49.573892   63168 start.go:125] hostinfo: {"hostname":"Nicks-MBP.home","uptime":5641601,"bootTime":1669664948,"procs":409,"os":"darwin","platform":"darwin","platformFamily":"Standalone Workstation","platformVersion":"12.6.1","kernelVersion":"21.6.0","kernelArch":"x86_64","virtualizationSystem":"","virtualizationRole":"","hostId":"2e97756e-e296-517d-96d1-91d34cd120f0"}
W0201 20:55:49.573997   63168 start.go:133] gopshost.Virtualization returned error: not implemented yet
I0201 20:55:49.579787   63168 out.go:177] 😄  minikube v0.0.0-unset on Darwin 12.6.1
I0201 20:55:49.588916   63168 notify.go:220] Checking for updates...
I0201 20:55:49.589715   63168 config.go:180] Loaded profile config "minikube": Driver=docker, ContainerRuntime=docker, KubernetesVersion=v1.26.1
...
```

File: cmd/minikube/cmd/logs.go [hunk 1]
```diff
@@ -150,4 +159,5 @@ func init() {
 	logsCmd.Flags().StringVar(&nodeName, "node", "", "The node to get logs from. Defaults to the primary control plane.")
 	logsCmd.Flags().StringVar(&fileOutput, "file", "", "If present, writes to the provided file instead of stdout.")
 	logsCmd.Flags().BoolVar(&auditLogs, "audit", false, "Show only the audit logs")
+	logsCmd.Flags().BoolVar(&lastStartOnly, "last-start-only", false, "Show logs from last start.")
```
Comment (spowelljr) [inline cmd/minikube/cmd/logs.go:162]:
```suggestion
	logsCmd.Flags().BoolVar(&lastStartOnly, "last-start-only", false, "Show only the last start logs.")
```

File: cmd/minikube/cmd/logs.go [hunk 2]
```diff
@@ -75,6 +77,13 @@ var logsCmd = &cobra.Command{
 				exit.Error(reason.Usage, "Failed to create file", err)
 			}
 		}
+		if lastStartOnly {
+			err := logs.OutputLastStart()
+			if err != nil {
+				klog.Errorf("failed to output last start logs: %V", err)
```
Comment (spowelljr) [inline cmd/minikube/cmd/logs.go:83]:
```suggestion
				klog.Errorf("failed to output last start logs: %v", err)
```

**Review by spowelljr — CHANGES_REQUESTED** [review]
Can you also run `make generate-docs` to update website docs for the new flag.

**Review by spowelljr — APPROVED** [review]

**Pull request merged**